### PR TITLE
feat(next): loosen peer deps to allow `next@canary`

### PR DESCRIPTION
**Pull request by mwskwong**
fixes https://github.com/vercel/geist-font/issues/76

### CODE REVIEW

File: packages/next/package.json [hunk 1]
```diff
@@ -53,6 +53,6 @@
   },
   "license": "SIL OPEN FONT LICENSE",
   "peerDependencies": {
-    "next": "^13.2 || ^14"
+    "next": ">=13.2.0 <15"
```
Comment (JohnPhamous) [inline packages/next/package.json:56]: Let's just allow anything newer than 13.2.0.
```suggestion
    "next": ">=13.2.0"
```

File: packages/next/pnpm-lock.yaml [hunk 2]
```diff
@@ -6,7 +6,7 @@ settings:
 
 dependencies:
   next:
-    specifier: ^13.2 || ^14
+    specifier: '>=13.2.0 <15'
```
Comment (JohnPhamous) [inline packages/next/pnpm-lock.yaml:9]:
```suggestion
    specifier: '>=13.2.0'
```

**Review by JohnPhamous — CHANGES_REQUESTED** [review]
Thanks for making the PR!

File: packages/next/pnpm-lock.yaml (hunk 2 above)
Comment (mwskwong) [reply packages/next/pnpm-lock.yaml:9]: This package is depending on `localFont` from `next`. We can't guarantee whether there will be any breaking changes of it in `> 14`. I'm not sure whether removing the version cap here is a good idea.

**Review by JohnPhamous — APPROVED** [review]

**Pull request merged**